apiary.apib

```diff
@@ -1196,6 +1251,13 @@ A business asset type group. Used to classify a business asset type and apply ba
 + `value`: General Maintenance (string, required) - The description of the purpose.
 + `isActive`: true (boolean, required) - Indicates if the purpose is active.
 
+## Trade Create (Base Audit Object)
++ `name`: Electrician (string, required) - The name of the trade.
++ `isActive`: true (boolean, required) - Indicates if the trade is active.
+
+## Trade (Trade Create)
```

Comment:
Trade should inherit from Trade Create but include the audit properties.  Since there isn't multiple inheritance we take the lesser of two evils.